adr/0020-minimum-supported-python-version copy.md

```diff
@@ -0,0 +1,32 @@
+# 0020. Minimum supported Python version
```

Comment:
` copy` seems like it should not be in the filename.
Needs adjustment in 0002 if changed

Reply:
We've discussed this, as this is a pretty different approach; this is no longer amending but fundamentally changing. Hence, a new ADR is created, and the old one is updated and marked as superseded, while a new one is introduced.

Reply:
Pushed a fix for the filename 👍 Thanks for spotting 👀

Reply:
It was the `%20copy` part of the link in 0002 I was referring to

Reply:
Fixed that one as well 👍